data-prepper-plugins/opensearch/src/test/java/com/amazon/dataprepper/plugins/sink/opensearch/BulkRetryStrategyTests.java

```diff
@@ -40,12 +47,18 @@ public class BulkRetryStrategyTests {
         setPipelineName(PIPELINE_NAME);
     }};
     private static final PluginMetrics PLUGIN_METRICS = PluginMetrics.fromPluginSetting(PLUGIN_SETTING);
+    private BiConsumer<DocWriteRequest<?>, Throwable> logFailureConsumer;
 
-    @Before
+    @BeforeEach
     public void metricsInit() {
         MetricsTestUtil.initMetrics();
```

Comment:
Suggest put it in the setUp instead of another @BeforeEach method

Reply:
Thanks for the suggestion @chenqi0805 ! I pushed a change for it. Please re-review.